projects/igniteui-angular/src/lib/carousel/carousel.component.ts

```diff
@@ -142,7 +146,7 @@ export class IgxCarouselComponent implements OnDestroy {
 
     /**
      * An event that is emitted after a slide has been added to the carousel.
-     * Provides a references to the IgxCarouselComponent and IgxSlideComponent as an event argument.
+     * Provides references to the IgxCarouselComponent and IgxSlideComponent as an event arguments.
```

Comment:
IgxCarouselComponent and IgxSlideComponent can be wrapped in `` as well to be highlighted in the output.